src/main/archetype/pom.xml

```diff
@@ -191,6 +191,43 @@
                     <groupId>biz.aQute.bnd</groupId>
                     <artifactId>bnd-maven-plugin</artifactId>
                     <version>${bnd.version}</version>
+                    <executions>
+                        <execution>
+                            <id>bnd-process</id>
+                            <goals>
+                                <goal>bnd-process</goal>
+                            </goals>
+                            <configuration>
+                                <bnd><![CDATA[
+Bundle-Category: ${componentGroupName}
+Import-Package: javax.annotation;version=0.0.0,*
```

Comment:
This is bundle specific!

Reply:
Good catch, moved back to core bundle in https://github.com/adobe/aem-project-archetype/pull/342/commits/7d3649515034ad5bab65675cdf116ef812d1395c